Python/pythonrun.c

```diff
@@ -962,14 +962,21 @@ print_exception(PyObject *f, PyObject *value)
     }
     else {
         PyObject* moduleName;
-        const char *className;
+        PyObject* qualName;
```

Comment:
This can be moved down to line 971, where it's first used; C99 includes intermingled declarations.